Source/Widgets/InfoBox/InfoBoxDescription.css

```diff
@@ -0,0 +1,42 @@
+body {
+    margin: 0;
+    padding: 0;
+}
+
+.cesium-infoBox-description {
+    font-family: sans-serif;
+    font-size: 13px;
+    padding: 4px 10px;
+    margin-right: 4px;
+    color: #edffff;
+}
+
+.cesium-infoBox-description table {
+    color: #edffff;
```

Comment:
That's a good point.  I guess it depends on what the goal is.  Should completely unstyled HTML look good by default?  I suppose the answer is yes.  That means that I'll have to apply a local style to the container div to make KML look good, I'll let you know if I have trouble figuring that out, otherwise I assume the issue resolved.